src/CpuAffinity.cc

```diff
@@ -22,7 +22,8 @@
 
 static CpuAffinitySet *TheCpuAffinitySet = NULL;
 
-void
+/// set CPU affinity for this process on startup
```

Comment:
This function documentation does not match this function usage -- the function is called during _reconfiguration_ as well, not just on startup.

```suggestion
/// Makes CPU affinity of this process match configured CPU affinity.
/// Assumes that we have never set the CPU affinity or have cleared it.
```